src/docs/contributing/approach/sdk-docs/write-performance.mdx

```diff
@@ -53,7 +59,11 @@ To develop content for this file, add code samples to these directories:
 - `/src/includes/performance/add-spans-example/`
 - `/src/includes/performance/retrieve-transaction/`
 
-Then determine whether to provide a code sample illustrating how to connect errors and spans. If so, add the SDK to `Supported` list in the `<PlatformSection>`, then add a code sample `/src/includes/performance/connect-errors-spans/`.
+If any of the above files are not applicable to your platform, create the file regardless and note that is not relevant for that SDK.
```

Comment:
I'd like to double check this. @bruno-garcia, I think you're probably most familiar with the structure; can you confirm?

Reply:
poked bruno in slack, hopefully we can get a response sometime next week 🤞

Reply:
You could alternatively just "ignore" the platform at the location where this include is used. I believe you've done exactly that?

i.e:
[src/platforms/common/performance/instrumentation/custom-instrumentation.mdx](https://raw.githubusercontent.com/getsentry/sentry-docs/fb675f93cf697f316554aca36a5d04dfdedbcbd3/src/platforms/common/performance/instrumentation/custom-instrumentation.mdx):

```
<PlatformSection notSupported={["java", "native"]}>

<PlatformContent includePath="performance/retrieve-transaction" />

</PlatformSection>
```

This way the advice to add the file even if the platform doesn't require it, could be removed.
But no strong feelings about the approach

Reply:
Looks like I'd gotten things mixed up while working on this, thanks for the clarification! I've updated the instructions to reflect this.